src/benchmarklib/benchmark_runner.cpp

```diff
@@ -463,6 +504,7 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
     ("r,runs", "Maximum number of runs of a single query (set)", cxxopts::value<size_t>()->default_value("10000")) // NOLINT
     ("c,chunk_size", "ChunkSize, default is 100,000", cxxopts::value<ChunkOffset>()->default_value("100000")) // NOLINT
     ("t,time", "Maximum seconds that a query (set) is run", cxxopts::value<size_t>()->default_value("60")) // NOLINT
+    ("w,warmup", "Number of seconds that a query is being warmed up", cxxopts::value<size_t>()->default_value("10")) // NOLINT
```

Comment:
"A query being warmed up" sound strange to my ears. ".. seconds that each query is run for warm up" ?